tests/sentry/tasks/test_delete_seer_grouping_records.py

```diff
@@ -33,50 +32,43 @@ def test_delete_seer_grouping_records_by_hash_batches(
             "args": [project_id, hashes, 100]
         }
 
-    @patch("sentry.tasks.delete_seer_grouping_records.logger")
```

Comment:
Testing the logger does not make sense. We need to test that the task got scheduled.

See this block:
https://github.com/getsentry/sentry/blob/282f05a21d591bcc4478f94dc20d4c3eb967a98b/src/sentry/tasks/delete_seer_grouping_records.py#L74-L79